8_5/Vision_test/main.py

```diff
@@ -0,0 +1,8 @@
+inputs = []
+counter = 0
+for i in range(3):
+    inputs.append(input())
+for x, y in zip(inputs[1],inputs[2]):
+        if x != y:
+            counter+=1
+print(counter)
```

Comment:
@maleki1999 
this is good and optimize answer
but for for variety in problem solving use 'set' this time